### PR TITLE
Need red cue for new file or link uploaded for the project #3274

### PR DESCRIPTION
**Pull request by suppermancool**
Need red cue for new file or link uploaded for the project #3274

### CODE REVIEW

File: src/projects/detail/containers/ProjectInfoContainer.js [hunk 1]
```diff
@@ -437,8 +438,10 @@ class ProjectInfoContainer extends React.Component {
     const canDeleteProject = currentMemberRole === PROJECT_ROLE_OWNER && project.status === 'draft'
 
     const projectNotReadNotifications = filterReadNotifications(filterNotificationsByProjectId(notifications, project.id))
+    console.log('totest projectNotReadNotifications', projectNotReadNotifications)
```
Comment (maxceem) [inline src/projects/detail/containers/ProjectInfoContainer.js]: `console.log` for debugging can be removed

File: src/routes/notifications/helpers/notifications.js [hunk 2]
```diff
@@ -262,6 +262,22 @@ export const filterTopicAndPostChangedNotifications = (notifications, tagRegExp)
...
+ * @param  {Array}  notifications list of notifications
+ *
+ * @return {Array}                notifications list filtered of notifications
+ */
+export const filterFileAndLinkChangedNotifications = (notifications) => {
+  return _.filter(notifications, (notification) => (
+    notification.eventType === EVENT_TYPE.TOPIC.CREATED ||
+    notification.eventType === EVENT_TYPE.POST.CREATED ||
+    notification.eventType === EVENT_TYPE.POST.UPDATED ||
+    notification.eventType === EVENT_TYPE.POST.MENTION
```
Comment (maxceem) [inline src/routes/notifications/helpers/notifications.js]: We should count notifications regarding file and links upload, with the next types https://github.com/appirio-tech/connect-app/blob/feature/workstreams/src/config/constants.js#L1050-L1051

image

Current implementation counts notifications about posts.

Comment (suppermancool) [reply src/routes/notifications/helpers/notifications.js]: ah sorry my mistake. Done in the latest commit

File: src/projects/detail/containers/ProjectInfoContainer.js (hunk 1 above)
Comment (suppermancool) [reply src/projects/detail/containers/ProjectInfoContainer.js]: my bad. done in latest commit

**Pull request merged**